content-jumpbox-provisioner/README.md

```diff
@@ -0,0 +1,35 @@
+# USE https://github.com/Financial-Times/upp-global-configs FOR GLOBAL CONFIGS
```

Comment:
Is this needed for the jumpbox provisioner?

Reply:
done. did a shitty job at copying and pasting